src/testIntegration/resources/application-test.yml

```diff
@@ -29,10 +29,6 @@ spring:
     redis:
       port: 6380
 
-mailcatcher:
-  api:
-    port: 1080
-
 app:
   jwt:
     private-key: 'MIIG/AIBADANBgkqhkiG9w0BAQEFAASCBuYwggbiAgEAAoIBgQCcp/b/R9+J1Wuj8pddehsf3y2bepEA4uaT+MEH8JTD/XchO93+XQQKJPwOr5pQBSscr6tEOTyIK5kwPclhBFbU1QTD3NcbjTwM0dmjVqUOj0Sg/gogvqBXyHYB3JXQWtreC4Ia64zMX5pFsgQc0PH81YSM5deJtOsjFc/FsSIjjJ9yMvJSK6wT1PtkeVpMQM0DjNoNxj3wsmzkHXuX+HdDxFjAsaVl7UcmSGL8jF5jNfr45Bl2jMVZFKYkMzAnigB1c6cTmMuXerojMphYnGS3zPwhNxfNJYswoSqRjEFLh47l2xYxErzH2eO06uPwjSOmk0kgO3R3CN+V1eFSYshVAAFbtN0CEmwCr9FjYrU2OhC+xCnNOGOuQcZGQogAKUrQ3hdxzAo2JOYj/AxDl5z/qw0lgRmSjjAhQAZvi7WQ0m2q6nAStq7qMvx9+SRIFA3OhDIjlatNpGfA6uqSAc79bQaKww1z5ELXP8oW073JtEVEHbX/hwcJ4fs26B0QcjsCAwEAAQKCAYBmgBxD4kIoTIV2q2dQ2XFgBEYyIhs4ij42eZ/L0yVzq6fEDgjtwuy+un5WGZ4NV6Riy6pzJEQwEN7Vyse1/AHYllzczuXMeDffZ3tQvYOYKDPOZwo3bKM7bMWR6EP6JUhp2vvRI2hn3PHzQkwgkrKTLR3ilwzVZJQpy7676m+c9Qjqztzw2PRpsYSftCtaE1hRR1rxglHK6OQIyiWxRPi8VX+A7Xt8WqmgKeLivfGQVqSVL6JdiDQkWjdvHjMW29r04pKwuxzMi+GvDklFaNr2AcIhPGsFmF5zCqZVW1p0psh8UjJOosI+aLHqWRU0gp9OKS4/J3pJa5NHEeF/uG92/UiZzX19VH9znvCy14Pe4l4Mknrf5BNsAw+K1rTmdNtJ5LX71Pe+Kh6unijzllUsQLuKdbq0PyP+SEVI9FG/ZQeBg9HNMGIMC3YgcTKqeAZaHa/szzVpggWgDzA1RaHnuW00Xjo6Z2EqRu+2SslznG9VZypZe/Cc3N/G6ngZP8kCgcEAzAF4+WzBwzrmVvvc1+yq6jsu+GJORIpjz+NIDcnnRYWrYtdL4DMlMvQwlDCo3u8zfADpXE/Y0+rm7ZmuawCO0qE6FWE0HiN7bFREcbXxqR5LAVhht6G+Eckal/UP6fvh6QNuupiZJZ3QNrVPDVFsI16do+0mDLrHKNIy//+Tdnw8LMiBEmfb0Jetk+on5jeVCdPx+MK7kUCX5MvhNcIVYV6uD2XTUtXYxkveCAPOHuOlI9LXrj7UKofQyS1KalmNAoHBAMSVIAXsRQGBACSwhMiL84chjW3/GFyOl7gWnu2b/U8miCVoLaDGj4JZS4q8qEAzs/K2pTr4TzvNCB8iB2WUAehz81U3KDtUErSKJoqw33n9uEQW35kVjjbXTEZVWd8Wa7mavq6ZM/kALSC9Hvp4jKDHND8na+VuNHrFSgXNj3IAbDv+rcXXrgF8G6NxfYoZ1qt1f+HaC6ElaaEAV9hm9wDHsmvbg1p/AVw6Yt2cRG1UmtZySK5plqHsgsAOA8005wKBwCGkvIal4O6JcCM4NhuF9EncqMpdq/qxt/Q4z1E20uUsPi92whSfWqskij5mJ/NcD2mL3PBUWrXYL21Pwuvyk5s465NLDIVL5LbgzsjR8KCrVkNKiBL4oZfdSn109MLpvz/mSQSM1fTWKdhb3kWb//F1URoPqg6/XsjkoXSGeZM4udPI3UBj0GR+Hyodm6bj07hjM6yKsNQriuE4nlN3Rn9gVeRELq2ihrVtHZjEx+FgOYPG6brasbKx40TKK1OOoQKBwHphF9l4GBFynIM6vL8YmRez5JboquLcBkmdE316kbyXkUQwUc94LVkJfAS3o8qsJUdhOwW2FaV146t2BMLQ3Es4+/A98TbWBPsN0iafK9CHChKhZrundqxfZeWXNuaxz4hxdGm/iGJoIK/z7HgjWfCxcq4qPXpMHDGd5pIsQTPIXZkP2GCE/5d/Crm6alvx7/k3odAZbQn5/4Mycn2OxBBhQYrx0hkhXdhrpKavpM2nUOlVpjemFhdh5JfpYaZ7WwKBwBpClMB7+ObSwpiS0sC0qLrQ1wbtyTOESypp9zM9Tto4dNnR1LA6T/p1AGt7pyFHK40j5ekTnTqUKsV4Qh5cFjeQpLyoYxqBfYncfbnevW/fFQU0aP6lOktUo7nLO6WXYyx+MsXB+fnPtk08XkR/c/uom913xYXihvERLReIGypxkTFG1iu1XlJw4sGyeeRaU//bawGFlzN06H2kzNMrZ5LwHmVmsoUdnmMtwNlLBCG9iGsWt56672nuv2vfPd6xEg=='
```

Comment:
not apart of the PR, but is this supposed to be here?

Reply:
not sure. will look into it as part of  a different PR